Content.Client/DragDrop/DragDropSystem.cs

```diff
@@ -242,7 +242,7 @@ private bool OnContinueDrag(float frameTime)
             if (_targetRecheckTime > TargetRecheckInterval)
             {
                 HighlightTargets();
-                _targetRecheckTime = 0;
+                _targetRecheckTime -= frameTime;
```

Comment:
shouldnt this detract `TargetRecheckInterval`?

Reply:
A